coordinator/src/services/validation/claim-validator.ts

```diff
@@ -14,7 +16,12 @@ export function verifyClaim(
   }
 
   if (claim.payee?.party) {
```

Comment:
after discussions I think we want to update the type of claim to have `payee` be mandatory